frame/evm/src/lib.rs

```diff
@@ -940,17 +943,34 @@ where
 		corrected_fee: U256,
 		base_fee: U256,
 		already_withdrawn: Self::LiquidityInfo,
+		target: Option<H160>
 	) -> Self::LiquidityInfo {
 		if let Some(paid) = already_withdrawn {
 			let account_id = T::AddressMapping::into_account_id(*who);
 
 			// Calculate how much refund we should return
 			let refund_amount = paid.peek().saturating_sub(corrected_fee.unique_saturated_into());
+			let caller_fees = corrected_fee / 2;
```

Comment:
Can we add some comments here? Our purpose that sharing 50% fee for contract deployer from the caller transaction fee

Reply:
@dung5ire Please check